.github/ISSUE_TEMPLATE/onboarding_to_cluster.yaml

```diff
@@ -0,0 +1,137 @@
+name: Onboard to a cluster
+description: Get access to an Operate-First managed cluster.
+title: "NEW PROJECT: <name>"
+labels: [kind/onboarding, area/cluster]
+assignees: [first-operator]
+body:
+    - type: dropdown
+      id: cluster
+      attributes:
+          label: Target cluster
+          description: |
+              Please select a cluster from the list at
+              https://github.com/operate-first/apps/tree/master/cluster-scope/overlays/moc#available-clusters
+          options:
+              - Zero
+              - Infra
+              - Rick
+    - type: input
+      id: team-name
+      attributes:
+          label:
```

Comment:
looks like you're missing the label here? resulting in: 

![image](https://user-images.githubusercontent.com/10904967/123969329-1d93c280-d986-11eb-92ee-315bb0cd5cd2.png)

Reply:
resolves, thanks :+1: